From e30d5146593f02fe17182cccf940962ba85b0c83 Mon Sep 17 00:00:00 2001
From: Igor Grinberg <igrinberg@habana.ai>
Date: Wed, 22 Sep 2021 17:32:54 +0300
Subject: [PATCH] [SW-55215] gaudi: use the map register name

The COLD_RST_FLOPS_2 got its new name - COLD_RST_DATA.
Use the new name where appropriate.

Change-Id: I49a061b171f5090de2ef131c67036c10490e0e37
Signed-off-by: Igor Grinberg <igrinberg@habana.ai>
---

diff --git a/arch/gaudi/arm/common.c b/arch/gaudi/arm/common.c
index c3958df..a769327 100644
--- a/arch/gaudi/arm/common.c
+++ b/arch/gaudi/arm/common.c
@@ -508,15 +508,14 @@
 	struct gaudi_flops_2_data flops_2_data;
 	u32 *data = (u32 *)(HL_SP_SRAM_BASE);
 
-	flops_2_data.data = regmap_readl(mmPSOC_GLOBAL_CONF_COLD_RST_FLOPS_2);
+	flops_2_data.data = regmap_readl(mmCOLD_RST_DATA);
 	if (flops_2_data.spsram_init_done != SPSRAM_INIT_DONE) {
 		/* SP SRAM access is limited to 32bit */
 		for (i = 0; i < HL_SP_SRAM_SIZE / sizeof(u32); i++)
 			data[i] = RESET_PATTERN;
 
 		flops_2_data.spsram_init_done = SPSRAM_INIT_DONE;
-		regmap_writel(flops_2_data.data,
-			      mmPSOC_GLOBAL_CONF_COLD_RST_FLOPS_2);
+		regmap_writel(flops_2_data.data, mmCOLD_RST_DATA);
 	}
 
 	cpu_boot_dev_sts_clrset(CPU_BOOT_DEV_STS0_SP_SRAM_EN, true);
